### PR TITLE
AT_032.001 

### PR DESCRIPTION
**Pull request by GettingPro**

### CODE REVIEW

File: src/test/java/GroupBugHuntersEditBuildInformationTest.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        createFreestyleProject();
+        getDriver().findElement(By.xpath("//a[@href='job/NewFreestyleProject/']")).click();
+        getDriver().findElement(By.xpath("//a[@href='lastBuild/']")).click();
+        getDriver().findElement(By.xpath("//a[@title='Edit Build Information']")).click();
+        getDriver().findElement(By.xpath("//input[@name='displayName']")).sendKeys("New build 123");
+        getDriver().findElement(By.xpath("//textarea[@name='description']")).sendKeys("Build 123 description test");
+        getDriver().findElement(By.xpath("//button[@type='submit']")).submit();
+        String buildName = getDriver().findElement(By.xpath("//span[@class='jenkins-icon-adjacent']")).getText();
+        String buildDescription = getDriver().findElement(By.id("description")).getText();
+        Assert.assertEquals(getDriver().getCurrentUrl(),"http://localhost:8080/job/NewFreestyleProject/lastBuild/");
```
Comment (RusAli) [inline src/test/java/GroupBugHuntersEditBuildInformationTest.java:32]: between 31,32 and 23,24 rows it will be nice to see white space)

**Review by RusAli — APPROVED** [review]

File: src/test/java/GroupBugHuntersEditBuildInformationTest.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+    public void createFreestyleProject() throws InterruptedException {
+        getDriver().findElement(By.xpath("//a[@title='New Item']")).click();
+        getDriver().findElement(By.xpath("//input[@id='name']")).sendKeys("NewFreestyleProject");
+        getDriver().findElement(By.xpath("//li[contains(@class,'hudson_model_FreeStyleProject')]")).click();
+        getDriver().findElement(By.xpath("//button[@type='submit']")).submit();
+        getDriver().findElement(By.xpath("//button[@type='submit']")).submit();
+        getDriver().findElement(By.xpath("//a[@title='Build Now']")).click();
+        getDriver().findElement(By.xpath("//a[@title='Back to Dashboard']")).click();
+        Thread.sleep(2000);
```
Comment (anonymoustuna) [inline src/test/java/GroupBugHuntersEditBuildInformationTest.java:16]: Is there any way to use it without Thread.sleep()?!

**Review by anonymoustuna — APPROVED** [review]

**Comment by LokDar** [comment]
[AT_032.001 | Build History > Edit build information > Rename Build and Change Description](https://trello.com/c/Rkg5e9YR/244-at032001-build-history-edit-build-information-rename-build-and-change-description)

**Comment by LokDar** [comment]
[AT_032.002 | Build History > Edit build information > Rename Build and Change Description > Verify Change on Project Status Page](https://trello.com/c/0tcGGDiN/341-at032002-build-history-edit-build-information-rename-build-and-change-description-verify-change-on-project-status-page)

**Comment by GettingPro** [comment]
[AT_032.003 | Build History > Edit build information > Rename Build and Change Description > Verify Change on Build History Page](https://trello.com/c/FiE7aOD4/342-at032003-build-history-edit-build-information-rename-build-and-change-description-verify-change-on-build-history-page)

**Pull request merged**